rosdep/python.yaml

```diff
@@ -3893,6 +3897,8 @@ python-trep:
   ubuntu:
     pip:
       packages: [trep]
+python-tshark:
+  ubuntu: [tshark]
```

Comment:
Please contribute at least debian rules too. Fedora macOS and Gentoo would be appreciated as well.